src/components/core/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState, ReactNode, HtmlHTMLAttributes } from 'react';
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox';
+import classNames from 'classnames';
+import css from './checkbox.module.scss';
+import CheckIconSVG from '../../../images/svg/checkbox-icon.svg';
+import CheckIndetermineIconSVG from '../../../images/svg/checkbox-indetermine-icon.svg';
+
+export const CheckBox = CheckboxPrimitive.Root;
+
+export interface CheckboxProps extends HtmlHTMLAttributes<HTMLInputElement> {
+    dark?: boolean;
+    indetermine?: boolean;
+    check?: boolean;
+    size?: 'default' | 'small';
+}
+
+const Checkbox = ({ children, dark, indetermine, check, size = 'default', ...props }: CheckboxProps) => {
+    return (
+        <div className={classNames(css.container)} {...props}>
```

Comment:
```suggestion
        <div className={css.container} {...props}>
```
We don't need to use `classNames()` here because there's only one css class

Reply:
Resolved in latest commit